browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -3423,18 +3412,13 @@ index 864ce3cfe13994642495f2d7dfbdcc9a4cad9f24..50a4c389709d92a251214b24ba6068d9
  #include "ScriptState.h"
  #include "SecurityOrigin.h"
  #include "UserGestureEmulationScope.h"
-@@ -103,9 +105,25 @@ void PageRuntimeAgent::didClearWindowObjectInWorld(Frame& frame)
+@@ -103,7 +105,22 @@ void PageRuntimeAgent::didClearWindowObjectInWorld(Frame& frame, DOMWrapperWorld
      if (!pageAgent)
          return;
  
--    auto frameId = pageAgent->frameId(&frame);
 +    // Force main world creation.
-     auto* scriptState = mainWorldExecState(&frame);
--    notifyContextCreated(frameId, scriptState, nullptr, true);
-+    Document* document = frame.document();
-+    String worldName = document ? document->securityOrigin().toRawString() : String();
-+    notifyContextCreated(frame, scriptState, worldName, true);
-+    frame.script().evaluateIgnoringException(ScriptSourceCode(pageAgent->bootstrapScript(), URL { URL(), "web-inspector://bootstrap.js"_s }));
++    auto* scriptState = mainWorldExecState(&frame);
+     notifyContextCreated(pageAgent->frameId(&frame), frame.script().globalObject(world), world);
```

Comment:
There is also a duplicate `notifyContextCreated` call in `PageRuntimeAgent::createIsolatedWorld`, please remove the latter.